test/winter_onboarding_2021/4clojure/alisha_test.clj

```diff
@@ -12,4 +12,9 @@
 
 (deftest vector-list-comparison
   (testing "comparison of vector & list"
-    (is (= (solutions/problem6) [:a :b :c]))))
```

Comment:
[Style] You can configure your editor to add a new line at the end of the file. This will avoid such diffs.